src/site/content/en/learn/html/dialog/index.md

```diff
@@ -3,8 +3,140 @@ title: 'Dialog'
+
+{% Codepen {
+user: 'web-dot-dev',
+id: 'BaOBLNy'
+} %}
+
+The `HTMLDialogElement` has three main methods, along with all the methods inherited from [`HTMLElement`](/learn/html/apis).
+
+```html
+dialog.show() /* opens the dialog */,
+dialog.showModal() /* opens the dialog as a modal */,
+dialog.close() /* closes the dialog */,
+```
+
+Because this `<dialog>` was opened via the [`HTMLDialogElement.showModal()`](https://developer.mozilla.org/docs/Web/API/HTMLDialogElement/showModal)
+method, it is a modal dialog. Opening a modal dialog deactivates and obscures everything other than the dialog itself. If you
+hover over the UI outside of the dialog, you’ll note all the elements are behaving as if [`pointer-events: none;`](https://developer.mozilla.org/docs/Web/CSS/pointer-events)
+was set; even the button that opens the dialog doesn’t react to interactions.
+
+When the dialog is opened, focus moves into the dialog. Focus is set on the first element in the sequential keyboard navigation order within that dialog.
+If you hit the `tab` key repeatedly, you’ll note that only the content within the dialog can get focus while the modal dialog is
+open. Everything outside of the modal dialog is inert as long as the dialog is open.
+
+When a dialog is closed, modal or not, focus is returned to the element that opened the dialog. If you programmatically
+open a dialog not based on user action, reconsider. If you must, ensure that focus is put back where it was prior to the dialog opening,
+especially if the user dismisses the dialog without interacting with it.
+
+There is a global `inert` attribute that can be used to disable an element and all of its descendants, other than any active
+dialog. When a modal dialog is opened using `openModal()`, the inertness or deactivation comes for free; the attribute
+isn’t explicitly set.
+
+The backdrop that obscures everything other than the dialog can be styled using the [`::backdrop`](https://developer.mozilla.org/docs/Web/CSS/::backdrop)
+pseudo-element. The backdrop is only displayed when a `<dialog>` is displayed with the `.showModal()` method. This pseudo-element
+matches all the backdrops, including the one displayed when the [FullScreen API](https://developer.mozilla.org/docs/Web/API/Fullscreen_API) is used,
+such as when viewing a video in full-screen mode which doesn’t have the same aspect ratio as the screen or monitor.
+
+## Non-modal dialogs
+
+The `HTMLDialogElement.show()` similarly opens a dialog, but without adding a backdrop or causing anything to become inert.
+The escape key does not close non-modal dialogs. Because of this, it is even more important to ensure you include a method
+of closing the non-modal dialog. In doing so, if the closer is outside the dialog, realize the focus will go to the element
+that opened the dialog, which may not be the best user experience.
+
+{% Codepen {
+user: 'web-dot-dev',
+id: 'bGKQvza'
```

Comment:
there is a typo in this codepen. "**not** that focus was" should read "**note** that focus was"